internal/controllers/machinedeployment/machinedeployment_rollout_planner_test.go

```diff
@@ -700,10 +849,27 @@ func (r rolloutScope) String() string {
 func msLog(ms *clusterv1.MachineSet, machines []*clusterv1.Machine) string {
 	sb := strings.Builder{}
 	machineNames := []string{}
+	acknowledgeMoveMachines := sets.Set[string]{}
+	if replicaNames, ok := ms.Annotations[clusterv1.AcknowledgedMoveAnnotation]; ok && replicaNames != "" {
+		acknowledgeMoveMachines.Insert(strings.Split(replicaNames, ",")...)
+	}
 	for _, m := range machines {
-		machineNames = append(machineNames, m.Name)
+		name := m.Name
+		if _, ok := m.Annotations[clusterv1.PendingAcknowledgeMoveAnnotation]; ok && !acknowledgeMoveMachines.Has(name) {
+			name += "🟠"
```

Comment:
Up to you, but maybe these icons are easier to interpret at a first glance: 
* move: →
* update: ⟳